### DISCUSSION TITLE
add support for 'hardhat-deploy' plugin

### DISCUSSION THREAD

**Discussion by SvenMeyer** [PaulRBerg/hardhat-template#36] [Ideas]
Would be a great addition !

https://hardhat.org/plugins/hardhat-deploy.html

**Comment by PaulRBerg** [comment]
Hey Sven, thanks for the suggestion!

I find `hardhat-deploy` sort of heavy-duty and unnecessary for most use cases. I've always been able to get by with [much simpler approaches](https://github.com/MainframeHQ/mainframe-lending-protocol/tree/29f6558a73315b5d5205bd52e490e851a47ba43c/scripts/deploy).

That said, I'll leave this issue open for discussion.

**Comment by SvenMeyer** [comment]
@paulrberg Thanks for the feedback. I am pretty new to hardhat and thought that the a dedicated deploy plugin would have "some benefits". Happy to check out "much simpler approaches" if that's what a day to day hardhat user recommends :-)
